inc/class-upgrade.php

```diff
@@ -279,4 +284,40 @@ private function upgrade_47() {
 			)
 		);
 	}
+
+	/**
+	 * Removes the 'wpseo-dismiss-about' notice for every user that still has it.
+	 */
+	private function upgrade_49() {
+		global $wpdb;
+
+		$usermetas = $wpdb->get_results( '
+			SELECT user_id, meta_value 
+			FROM ' . $wpdb->usermeta . ' 
+			WHERE meta_key = "wp_yoast_notifications" && meta_value LIKE "%wpseo-dismiss-about%"
+			', ARRAY_A
+		);
+
+		foreach ( $usermetas as $usermeta ) {
+			$notifications = unserialize( $usermeta['meta_value'] );
+
+			foreach ( $notifications as $notification_key => $notification ) {
+				if ( ! empty( $notification['options']['id'] ) && $notification['options']['id'] === 'wpseo-dismiss-about' ) {
+					unset( $notifications[ $notification_key ] );
+				}
+			}
+
+			// Using a hard query, because WordPress is caching the user meta.
```

Comment:
Logic suggests that this will bypass updating the cache; thus on object-cache enabled environments it will not be removed until the cache expires and is re-read.

Any way to use `update_user_meta` to save the changes?